src/goodbase/goodbase_cli.py

```diff
@@ -492,7 +492,15 @@ def dependencies(binder: inject.Binder) -> None:
                 for module, errmsg in revision.errors.items():
                     click.echo(click.style(f"\t{module}: {errmsg}", fg="yellow"))
         else:
-            click.echo(click.style("No revision found", fg="red"))
+            click.echo(
+                click.style(
+                    "No revision found\n"
+                    "For more detailed information about what command is doing, use `--verbose` option\n"
+                    "For more information about how to specify criteria, see"
+                    " https://dbradf.github.io/git-co-evg-base/concepts/criteria/",
```

Comment:
I wonder if we would want to wait for [DAG-1687](https://jira.mongodb.org/browse/DAG-1687) before doing this, since the URL would change with that migration.

Reply:
We have a couple more references to this wiki, which should be updated while moving the code to the new repo.
I think it's doesn't really matter to do it before or after [DAG-1687](https://jira.mongodb.org/browse/DAG-1687), but I'm fine to wait.